yarn2nix/src/Distribution/Nodejs/Package.hs

```diff
@@ -190,3 +192,16 @@ formatWarning = \case
          Nothing  -> "Leaving it out")
     <> "."
   (PlainWarning t) -> t
+
+parsePackageKeyName :: Text -> YLT.PackageKeyName
```

Comment:
Since this is now exposed, could you document what it does with a haddock comment?

Reply:
Done